### DISCUSSION TITLE
Scrollbars and buttons for the number of creatures, add the ability to left-click and hold to continuously scroll the scrollbar or increase/decrease the number of creatures [Done]

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#2687] [Ideas]
In fheroes2, it would be nice to add the ability to left-click and hold to continuously scroll the scrollbar or increase/decrease the number of creatures in addition to operating multiple clicks in a row.

In fheroes2:

![Mouse left button maintained fh2 01](https://user-images.githubusercontent.com/43583994/106063022-dfa12280-60f7-11eb-9ec0-05361e387a75.png)

![Separation slots fh2 avec boutons 3 4 5 01](https://user-images.githubusercontent.com/43583994/106055462-b24f7700-60ed-11eb-85b7-9d6b6ae775f7.png)

![Recruit window mouse wheel fh2 01](https://user-images.githubusercontent.com/43583994/106059263-c0ec5d00-60f2-11eb-9e34-053de557108b.png)

![Marketplace window mouse wheel fh2 01](https://user-images.githubusercontent.com/43583994/106059284-c9dd2e80-60f2-11eb-8a43-7ea8f6599f64.png)

**Comment by Branikolog** [comment]
This would be a nice addition to our list of issues. ;)
Don't forget to mention hero/town list and "kingdom overview" window as well.

**Reply by LeHerosInconnu** [reply]
@Branikolog.

> This would be a nice addition to our list of issues. ;)
> Don't forget to mention hero/town list and "kingdom overview" window as well.

Done here: https://github.com/ihhub/fheroes2/issues/2809.

**Comment by LeHerosInconnu** [comment]
This has been implemented.
I close the discussion.
